wordpress-mobile/WordPress-iOS
# Aztec: Dismiss Button

### Details:
In this PR we're matching the Aztec's Dismiss Button UX, with the Hybrid Editor:

- When we're being Modally Displayed, we'll render an `x`
- When the Editor is within a navigationController, we'll render, instead, the classic `<` chevron

### To test:
1. Press the `+` button to create a new post
2. Verify that the dismiss button shows an `X`
3. Dismiss the editor. Open any random Site, and edit a Post
4. Verify that the back button is rendered as the `<` chevron

Needs review: @astralbodies
Thanks Aaron!!
